ui/app/registration/controllers/navigationController.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
 
 angular.module('bahmni.registration')
-    .controller('NavigationController', ['$scope', '$rootScope', '$location', 'sessionService', '$window', 'appService', '$sce', 'offlineService', 'schedulerService',
-        function ($scope, $rootScope, $location, sessionService, $window, appService, $sce, offlineService, schedulerService) {
+    .controller('NavigationController', ['$scope', '$rootScope', '$location', 'sessionService', '$window', 'appService', '$sce', 'offlineService', 'schedulerService', '$http',
+        function ($scope, $rootScope, $location, sessionService, $window, appService, $sce, offlineService, schedulerService, $http) {
             $scope.extensions = appService.getAppDescriptor().getExtensions("org.bahmni.registration.navigation", "link");
             $scope.isOfflineApp = offlineService.isOfflineApp();
+            $scope.isSelectiveSyncStrategy = false;
+
+            var verifySelectiveSync = function () {
+                $http.get('/openmrs/ws/rest/v1/eventlog/filter/globalProperty/', {
+                    method: "GET",
+                    params: { q: 'bahmniOfflineSync.strategy' },
```

Comment:
@gopikrishna-yaramothu same as mentioned above

Reply:
@binduak 
this has been fixed